src/nsidc/iceflow/itrf/converter.py

```diff
@@ -60,7 +62,7 @@ def transform_itrf(
     transformed_chunks = []
     for source_itrf, chunk in data.groupby(by="ITRF"):
         # If the source ITRF is the same as the target for this chunk, skip transformation.
-        if source_itrf == target_itrf:
+        if source_itrf == target_itrf and target_epoch is None:
```

Comment:
We only skip transforming a chunk if both the source and target ITRF match, and no target epoch is given.

This allows us to do plate-motion coordinate propagation within a single ITRF (e.g., IS2 is in ITRF2014, and we want to compare to other data in ITRF2014, but we want to be able to change the epoch to account for plate motion over time).